<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ID3 Editor (MP3  Tag Editor)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QSplitter" name="splitter_all">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeView" name="treeViewDir">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refresh">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_scanSubFolders">
            <property name="text">
             <string>&amp;Scan Sub Folders</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_files">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_goPath">
             <property name="text">
              <string>&amp;Go</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Path : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_path"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="CTableView" name="tableViewFiles">
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QGroupBox" name="groupBox_mp3Info">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>MP3 File Info</string>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>File Size:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>File Date:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Bitrate :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Sample :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Channels :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Length :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_channels">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>100</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Channels :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_fileDate">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>40</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>File Date:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_bitrate">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>60</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Bitrate :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_length">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>120</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Length :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_fileSize">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>20</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>File Size:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_sampleRate">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>80</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Sample Rate</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_table">
           <property name="title">
            <string>Table</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBox_showDir">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Show Directory in Table</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_editCell">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Edit Cell Mode</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_fileName">
           <property name="title">
            <string>Filename</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="pushButton_FilenameAutoFix">
                <property name="text">
                 <string>Auto Fix</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_FilenameCase">
                <property name="text">
                 <string>Case</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_folderFix">
                <property name="text">
                 <string>Folder Fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_artist">
           <property name="title">
            <string>Artist</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pushButton_setArtistFromParent">
                <property name="text">
                 <string>From Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_artistCase">
                <property name="text">
                 <string>Case</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_album">
           <property name="title">
            <string>Album</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="pushButton_setAlbumFromDirectory">
                <property name="text">
                 <string>From Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_setAlbumCase">
                <property name="text">
                 <string>Case</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_title">
           <property name="title">
            <string>Title</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="pushButton_setTitleAsFilename">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="text">
                 <string>From Filename</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_setTitleCase">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="text">
                 <string>Case</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_misc">
           <property name="title">
            <string>Misc</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton_setColumn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Set Column as Cell</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QPushButton" name="pushButton_setAutoTrackNumber">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Auto Track Num</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_YearFromDirectory">
                <property name="text">
                 <string>Year from Dir</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="pushButton_clearComments">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Clear Comments</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_AutoFixAll">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Auto Fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_revert">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Revert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonExit">
        <property name="text">
         <string>E&amp;xit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>ctableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
